src/modules/video_player/index.js

```diff
@@ -103,10 +103,43 @@ class VideoPlayerModule {
             defaultValue: false,
             description: 'Disables autoplay of recommended videos on VoDs'
         });
+        settings.add({
```

Comment:
remove this setting. we don't need more settings.

Reply:
Removed. It's enabled by default now.